src/libstd/build.rs

```diff
@@ -86,6 +86,7 @@ fn main() {
 
 fn build_libbacktrace(host: &str, target: &str) -> Result<(), ()> {
     let native = native_lib_boilerplate("libbacktrace", "libbacktrace", "backtrace", ".libs")?;
+    let cflags = env::var("CFLAGS").unwrap_or_default() + " -fvisibility=hidden -O2";
```

Comment:
Is this related to bitcode embedding?

Reply:
Sort of tangentially yeah, but it's otherwise fixing issues that @comex brought up when testing this

Reply:
I later saw that this is in a separate commit with proper commit message anyway. Thanks for the info!